rm/rm-client/src/main/java/org/ow2/proactive/resourcemanager/common/event/RMNodeEvent.java

```diff
@@ -191,11 +191,16 @@ public RMNodeEvent(final RMNodeDescriptor rmNode, final RMEventType eventType, f
         this.nodeLocker = rmNode.getLockedBy();
     }
 
+    @Override
+    public String getKey() {
```

Comment:
There is something I don't understand here. The upper class has a `getKey` implementation that relies on the counter. But in this class it is overriden to rely on the `nodeUrl`. For different events, the `nodeUrl` can be the same, whereas they will have different counters. 
1) I don't understand how we can build event identity with the nodeUrl that is not unque for different events.
2) I don't understand in which case the counter is used to identify an event.

Reply:
Actually I no need getKey for RMEvent, so I putted counter

We can have several events with the same nodeUrl but different counter
However, in SortedUniqueSet I want to have only latest event associated with each node/nodesource